test/e2e/node/container_probe.go

```diff
@@ -33,36 +31,23 @@ import (
 
 const (
 	probTestContainerName       = "test-webserver"
-	probTestInitialDelaySeconds = 30
+	probTestInitialDelaySeconds = 15
```

Comment:
This is OK for now, but in generally I like to avoid unrelated changes like this bundled in a larger PR (prefer to send a separate PR for it).